README.md

```diff
@@ -1,10 +1,12 @@
 [![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue) [![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![StackOverflow](https://img.shields.io/badge/StackOverflow-questions-blueviolet)](https://stackoverflow.com/questions/tagged/fairlearn)
 
-# Fairlearn
+# Fairlearn - Alpha release
 
-Fairlearn is a Python package that empowers developers of artificial intelligence (AI) systems to assess their system's fairness and mitigate any observed unfairness issues. Fairlearn contains mitigation algorithms as well as a Jupyter widget for model assessment. Besides the source code, this repository also contains Jupyter notebooks with examples of Fairlearn usage.
+Fairlearn is an open source toolkit that empowers data scientists and developers to assess and improve the fairness of their AI systems.  Fairlearn has three components: Python functions for computing fairness metrics, mitigation algorithms for optimizing fairness metrics, and an interactive visualization dashboard.
```

Comment:
Definitely appreciate you improving this! I was originally planning to move most of the content of the README into the website. Specifically, some of this should probably live in the new "About" section that is still pretty empty. I do realize that some people just come to the repo as opposed to the website, so perhaps we can have it in both places? What do you think?

Also @mesameki @MiroDudik @hannawallach may have thoughts on the location.

Reply:
@romanlutz My vote is to put off structural revisions to the marketing page / website / repo documentation for now.  I think what you've done is great to allow things to get started! and with that in place, focusing on the substance for a while will be more productive.

Then when there are more great examples and substance, along with a roadmap some kind of shared understanding of what audience the project is trying to reach and what it's positioning is... then it becomes more straightforward to critique whether the website serves those needs and rearrange it.